<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTcpEdgeTestDataView_AppInstallReq</class>
 <widget class="QWidget" name="CTcpEdgeTestDataView_AppInstallReq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTcpEdgeTestDataView_TimeReadReq</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>任务ID(JobId)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="_App_"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="_Policy_"/>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="_JobId_"/>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="__JobId__">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>应用名称(App)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>时间间隔(Policy)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QCheckBox" name="__Policy__">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>(备注:缺省或等于0时表示立即升级)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>容器名称(Container)</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>内存配置(CfgMem)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QCheckBox" name="__Container__">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="4">
    <widget class="QCheckBox" name="__CfgMem__">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="CTcpEdgeCtrl_LineEdit" name="_CfgCpu_">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="4">
    <widget class="QCheckBox" name="__CfgCpu__">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="CTcpEdgeCtrl_LineEdit" name="_CfgMem_">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>CPU配置(CfgCpu)</string>
     </property>
    </widget>
   </item>
   <item row="12" column="4">
    <widget class="QCheckBox" name="__Enable__">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QCheckBox" name="__App__">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="_Version_"/>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>应用版本号(Version)</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QCheckBox" name="__Version__">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="4">
    <widget class="QGroupBox" name="_File_">
     <property name="title">
      <string>文件信息(File)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="3">
       <widget class="QCheckBox" name="__Path__">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnName">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>文件大小(Size)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="__Name__">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="_Path_"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="_Size_"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="_FileType_"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>文件路径(Path)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="_Name_"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>文件类型(FileType)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="__FileType__">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="__Size__">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>文件名(Name)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>(备注:单位byte)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>使能(Enable)</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QComboBox" name="_Enable_">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="_Container_">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CTcpEdgeCtrl_LineEdit</class>
   <extends>QLineEdit</extends>
   <header>../Ctrl/TcpEdgeCtrl_LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../TcpEdgeServer/TcpEdgeServer.qrc"/>
 </resources>
 <connections/>
</ui>
